packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py

```diff
@@ -563,36 +608,194 @@ def clear_job_annotations(job_id: int) -> None:
             raise
 
 
-def update_job_assignee(id: str, assignee_id: int | None):
+def setup_gt_job(task_id: int, dataset_path: Path, format_name: str) -> None:
+    gt_job = get_gt_job(task_id)
+    upload_gt_annotations(gt_job.id, dataset_path, format_name=format_name)
+    finish_gt_job(gt_job.id)
+    settings = get_quality_control_settings(task_id)
+    update_quality_control_settings(settings.id)
+
+
+def get_gt_job(task_id: int) -> models.JobRead:
     logger = logging.getLogger("app")
 
     with get_api_client() as api_client:
         try:
-            api_client.jobs_api.partial_update(
-                id=id,
-                patched_job_write_request=models.PatchedJobWriteRequest(assignee=assignee_id),
+            (paginated_jobs, _) = api_client.jobs_api.list(task_id=task_id, type="ground_truth")
+            assert (
```

Comment:
I'd change this assert to an explicit check and exception, as it's not a developer (the caller) error.